### DISCUSSION TITLE
Styled select with API

### DISCUSSION THREAD

**Discussion by saloeric** [tasteui/tasteui#311] [General]
Good morning,

First of all, congratulations on the project. Now, the question is, I am using the styled select with API, but when it is an edit, the request(route) is not executed, and I have no way to select the saved value because there are no options. How can I solve this problem?

I apologize for the English; I used a translator.

Thank you in advance.

**Comment by devajmeireles** [comment]
Hey, @saloeric !

Sorry about being late. For some reason, I was not notified about your topic.

Unfortunately, I was not able to understand exactly. What do you meant exactly when you say "but when it is an edit, the request(route) is not executed"?

**Comment by saloeric** [comment]
Hello, I think it will be easier to explain this way:
How to set a value in the "styled API" select in a form edit? In other words, the value was saved during creation, and in the edit, I want it to be automatically selected.

Thank you.

**Comment by joelhmetz** [comment]
https://discord.gg/jycbaerx
